src/clr-angular/progress/spinner/spinner.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component, Input } from '@angular/core';
+
+export const SPINNER_BASE_CLASS = 'spinner';
+
+@Component({
+  selector: 'clr-spinner',
+  template: `<ng-content></ng-content>`,
+  host: {
+    class: SPINNER_BASE_CLASS,
+    '[attr.aria-live]': 'liveRegion',
+    '[attr.aria-busy]': 'true',
+  },
+})
+export class ClrSpinner {
+  @Input('liveRegion') liveRegion: string = 'polite';
```

Comment:
Instead of `string` type you can use a union type of `@Input('liveRegion') liveRegion: 'polite' | 'assertive' = 'polite';

Reply:
I like that :)